internal/pkg/errorconcept/provision_watcher.go

```diff
@@ -105,6 +106,20 @@ func (r provisionWatcherDeviceServiceNotFound_StatusNotFound) message(err error)
 	return "Device service not found"
 }
 
+type provisionWatcherNameNotProvided struct{}
```

Comment:
Could we use one of the common Error Concepts for this situation which is basically a `BAD REQUEST` due to missing necessary data, possibly `errorconcept.InvalidRequest_StatusBadRequest`?